docs/docs/core/useAnimatedStyle.mdx

```diff
@@ -58,6 +60,12 @@ Only relevant when using Reanimated [without the Babel plugin on the Web](https:
 
 `useAnimatedStyle` returns an animated style object which has to be passed to the `style` property of an [Animated component](/docs/fundamentals/glossary#animated-component) that you want to animate.
 
+Types of `useAnimatedStyle` are made so they mimic behavior of `StyleSheet` styles as much as possible - `updater` callback return value should just look like a regular style object - with the exception of using shared values.
+
+:::caution
```

Comment:
```suggestion

```

We had a tendency to overuse the `:::caution` and `:::note` tags in the past. I'd recommend avoiding using it. If everything is marked as important, nothing is.